akka-remote/src/main/resources/reference.conf

```diff
@@ -147,41 +180,41 @@ akka {
         compression {
           # possibility to disable compression by setting this to off
           enabled = on
```

Comment:
I think we should remove this, and only have it as programatic toggle for testing (if we need to turn it off at all). We could use a long `advertisement-interval` for such tests.